### PR TITLE
refactor: use 'contents' Pinia store, no more Vuex store modules

### PR DESCRIPTION
**Pull request by orlinmalkja**

### CODE REVIEW

File: src/components/ImageView.vue [hunk 1]
```diff
@@ -10,18 +10,18 @@
 <script setup>
 import OpenSeadragon from 'openseadragon';
 import { computed, ref, watch } from 'vue';
-import { useStore } from 'vuex';
+import { useContentsStore} from '@/stores/contents';
```
Comment (paulpestov) [inline src/components/ImageView.vue:13]: space

File: src/components/TreeView.vue [hunk 2]
```diff
@@ -47,16 +47,16 @@ import Tree from 'primevue/tree';
 import {
   computed, nextTick, onMounted, ref, watch,
 } from 'vue';
-import { useStore } from 'vuex';
 import { useConfigStore } from '@/stores/config';
+import { useContentsStore} from '@/stores/contents';
```
Comment (paulpestov) [inline src/components/TreeView.vue:51]: space

File: src/components/ImageView.vue (hunk 1 above)
Comment (orlinmalkja) [reply src/components/ImageView.vue]: removed

File: src/components/TreeView.vue (hunk 2 above)
Comment (orlinmalkja) [reply src/components/TreeView.vue]: added it

**Review by paulpestov — APPROVED** [review]

**Pull request merged**